### ISSUE TITLE
Add ErodeBlendBrush

### ISSUE THREAD

**Issue by StealWonders** [ervinnnc/VoxelSniper#7]
This doesn't seem to exist anymore. 
I could find it in the old FAVS Github repo so this might help: https://github.com/IntellectualSites/FastAsyncVoxelSniper/blob/master/src/main/java/com/thevoxelbox/voxelsniper/brush/ErodeBlendBrush.java

**Comment by ghost** [comment]
Hey, thanks for your submission!

Will look into the suitability of this brush. How does this brush differ from Erode brush and Blend brush?

**Comment by StealWonders** [comment]
A friend has made the following screenshot to provide you with the difference to the normal ErodeBrush.

<img width="1920" alt="2020-04-10_12 37 18" src="https://user-images.githubusercontent.com/6114858/78985406-fe163d00-7b28-11ea-8195-6b442f3aa91e.png">

> While both are quite similar, eb has more post-erode smoothing than b

This helps builders to more quickly make good looking terrain, which is what this plugin is all about (at least I hope so). I am not asking for a replacement of the old ErodeBrush but a new brush, so the old one stays and this one is added alongside it.

**Comment by Konvett** [comment]
@ervinnnc I have some light I can shed on this issue!

This ability was originally possible through a second plugin that hooked into VoxelSniper, called Better Brushes. There are hardly any links to it, but I found one through a YouTube video (I installed it, it's working): http://www.mediafire.com/file/zt4dhifhcv434zt/Guia_informacion_Plugin__%2528_Better_Brushes_%2529.rar/file

Please let me know if this gets added, it's a great feature!

**Comment by Picorims** [comment]
I agree on the fact this has always been a useful and time saving brush, and unfortunately BetterBrushes broke since the 1.15 version. If you ever attempt to add it to your rewrite, be sure the community will be very thankful for the return of such a feature!
